content/docs/guides/gke/troubleshooting-gke.md

```diff
@@ -24,6 +24,64 @@ Here are some tips for troubleshooting Cloud IAP.
 
  * Make sure you are using HTTPS
 
+### DNS name not registered
+
+This section provides troubleshooting information for problems creating a DNS entry for your ingress. This section assumes
+you are using Cloud Endpoints and the DNS name
+
+```
+https://${DEPLOYMENT_NAME}.endpoints.${PROJECT}.cloud.goog
+```
+
+Symptoms
+
+  * When you access the the URL in chrome you get the error: **server IP address could not be found**
```

Comment:
s/chrome/Chrome

Question: Does the error show up only in Chrome?

Reply:
No but you might get different error messages in different web browsers.